src/components/artwork_grid.tsx

```diff
@@ -66,10 +66,8 @@ export class ArtworkGrid extends React.Component<GridProps, null> {
       for (let j = 0; j < sectionedArtworks[i].length; j++) {
         const artwork = sectionedArtworks[i][j]
         artworkComponents.push(
-          <Artwork
-            artwork={artwork}
-            key={"artwork-" + j + "-" + artwork.__id}
-          />)
+          <Artwork artwork={artwork as any} key={"artwork-" + j + "-" + artwork.__id} />,
```

Comment:
I don't remember what the final reason was for me to go with one line, but it was the result of a bunch of linter faffing.

I am completely fine with the multi-line approach. Feel free to reintroduce it whenever.